dev_guide/pod_autoscaling.adoc

```diff
@@ -71,7 +74,7 @@ directly to the replica count of the deployment configuration. Note that autosca
 applies only to the latest deployment in the `Complete` phase.
 
 [[creating-a-hpa]]
-== Creating a Horizontal Pod Autoscaler
+== Autoscaling for CPU Utilization
```

Comment:
HPA V2 support both cpu and memory resource metrics, you need to describe customer can use both HPA v1 and v2 to create a hpa Autoscaling for cpu utilization in here.